plugins/inputs/http/http.go

```diff
@@ -64,6 +78,12 @@ var sampleConfig = `
   ## HTTP entity-body to send with POST/PUT requests.
   # body = ""
 
+  ## Cookie-based authentication
+  # cookie_auth_url = "https://localhost/authMe"
+  # cookie_auth_method = "POST" # default: POST
+  # cookie_auth_body = '{"username": "user", "password": "pa$$word", "authenticate": "me"}'
+  # cookie_auth_renewal = "8h" # default: "5m"
```

Comment:
Please skip the extra `default` comment as it currently breaks toml IIRC. The standard is to specify the default in the commented option like
```suggestion
  # cookie_auth_method = "POST"
  # cookie_auth_body = '{"username": "user", "password": "pa$$word", "authenticate": "me"}'
  # cookie_auth_renewal = "5m"
```